src/views/components/EntityCard.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+	<div class="entity-card" :style="getCardStyle()">
+		<img class="img-tier" 
+			:class="{'user-tier': !this.isOpponent(), 'opponent-tier': this.isOpponent()}" 
+			:src="getEntityTierPath()" />
+
+		<img class="img-entity" 
+			:class="{'user-pet': !this.isOpponent(), 'opponent-pet': this.isOpponent()}" 
+			:src="getEntityPath()" />
+
+		<img class="img-pedestal" src="../../assets/images//layout/shop-pedestal-with-shadow.png" />
+
+		<template v-if="isPet()">
+			<img class="img-stats" src="../../assets/images/layout/animal-stats-template.png" />
+			<span class="card-text text-attack">{{getPetAttack()}}</span>
+			<span class="card-text text-health">{{getPetHealth()}}</span>
+		</template>
+	</div>
+</template>
+
+<script>
+export default {
+  name: 'EntityCard',
+  props: {
+	cardWidth: Number,
+	cardHeight: Number,
+
+	entityType: String, /* Expected values:		"Pet", "Item" */
```

Comment:
you should put those comments in the declaration of the prop like
props:{
    cardWidth: {
        type: Number,
        required: true,
        default: 55,
        description: 'this is a card width'

    }
}

Reply:
Nice.